air/src/execution_step/boxed_value/stream.rs

```diff
@@ -29,9 +28,8 @@ use std::fmt::Formatter;
 /// of values that interpreter obtained from one particle. It means that number of generation on
 /// a peer is equal to number of the interpreter runs in context of one particle. And each set of
 /// obtained values from a current_data that were not present in prev_data becomes a new generation.
-// TODO: make it non-pub after boxed value refactoring.
 #[derive(Debug, Default, Clone)]
-pub(crate) struct Stream(Vec<Vec<ValueAggregate>>);
```

Comment:
It's because `ExecutionError` became public and `Stream` is used inside one of it's variants. And `ExecutionError` became public to use it in tests.

Btw, after boxed value landed, `Stream`'ll in a separate crate and must be public anyway, so it's all ok.